.github/workflows/ci.yml

```diff
@@ -23,7 +23,10 @@ jobs:
         features: [ "" , "--features alloc,simd,simd-nightly"]
 
     # Prevents a workflow run from failing when a job fails
-    continue-on-error: ${{ matrix.channel != 'stable' }}
+    # If target is x86 or x86_64, 
+    # or we are using simd-nightly while not on the nightly channel
+    # do not continue on error
+    continue-on-error: ${{ !contains(matrix.target, 'x86') || (contains(matrix.features, 'simd-nightly') && matrix.channel != 'nightly') }}
```

Comment:
I think the "x86" check will need to be either an exact comparison against the various targets or at least a check for the substrings `x86_64` and `i686`.